python/marvin/web/templates/header.html

```diff
@@ -1,6 +1,35 @@
 
 {% from 'macros.html' import galidsearch, login %}
 
+
+<!-- Begin Cookie Consent plugin by Silktide - http://silktide.com/cookieconsent.
+We use it to show a banner if the used browser does not play well with Marvin.-->
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+<script>
+window.addEventListener("load", function(){
+    if (navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
+        window.cookieconsent.initialise({
+          "palette": {
+            "popup": {
+              "background": "#000"
+            },
+            "button": {
+              "background": "#f1d600"
+            }
+          },
+          "position": "top",
+          "cookie": {
+              "expiryDays": 1},
```

Comment:
Does this mean a cookie is created which expires after 1 day?  Does this mean the banner only displays at most once a day?  Is this the behavior we want?  That seems reasonable to me, but just want to double-check.